### DISCUSSION TITLE
font-awsome icon support

### DISCUSSION THREAD

**Discussion by pa-terrythibodeau** [retypeapp/retype#182] [Ideas]
[Font-awesome](https://fontawesome.com/v5.15/icons?d=gallery&p=2) support would be amazing. It's pretty standard for web icons.
The Octicons library is very limited.

**Comment by geoffreymcgill** [comment]
Thanks for the suggestion.

We have a goal to support configuring any custom icon set and supporting Fontello, which itself support font-awesome. See https://github.com/retypeapp/retype/discussions/109

With some luck, we might see this in Retype `v1.12`.

I'll keep this thread updated with our progress.

**Comment by geoffreymcgill** [comment]
Hi @pa-terrythibodeau, 

Custom `svg` icons are now supported in Retype `v4.4.0`. See https://retype.com/blog/2026-03-23-whats-new-in-retype-v440/#custom-icons and https://retype.com/components/icon/.

Hope this helps.
